internal/provider/iam/data_group_test.go

```diff
@@ -0,0 +1,54 @@
+package iam_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-hcp/internal/provider/acctest"
+)
+
+// TODO : update the below tests to use a created group resource via Terraform
+func TestAccGroupDataSource(t *testing.T) {
+	dataSourceAddress := "data.hcp_group.test"
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGroupConfig("jolisa-group-test-2"),
```

Comment:
Instead of using a local group, we can create a group in the integration environment using the E2E account (1Password name screenshot below). This way the test won't be dependant on the create group functionality being implemented in the provider. I feel long term we should be able to just keep the group resource in the int account and use it for E2E tests. The E2E tests should already be set up to use the integration account within GitHub Actions. Open to some more thoughts on this!
<img width="471" alt="Screenshot 2024-02-21 at 10 30 31 AM" src="https://github.com/hashicorp/terraform-provider-hcp/assets/15036233/e33f941c-1070-42ff-a1fb-3b8901ea8f05">

Reply:
@insanejasper I've updated the test coverage to point to a group in the e2e tooling account. Let me know if that aligns with what you'd had in mind!